### ISSUE TITLE
Implement the standardised E1.31 Extended Discovery packets

### ISSUE THREAD

**Issue by peternewman** [OpenLightingProject/ola#1139]
Change to Root E1.31 Extended vector and the new E1.31 extended discovery vector. Add an option to implement one or the other discovery standards.

**Comment by nomis52** [comment]
We should remove the draft implementation and implement the approved version.

The draft was there for testing.

**Comment by peternewman** [comment]
Andrew seems to agree with you:
https://groups.google.com/forum/#!topic/open-lighting/RZo3fDciB4s

You don't think we should leave the draft in, like the rev2 packets?

**Comment by nomis52** [comment]
No. The only reason I left draft sACN in was because some consoles had implemented it. To the best of my knowledge no one implemented draft discovery.

**Comment by peternewman** [comment]
No-one implemented draft discovery aside from OLA you mean :)
